src/vl-wizard.js

```diff
@@ -0,0 +1,124 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-progress-bar/dist/vl-progress-bar-all.js';
+import '/lib/wizard.js';
+import './vl-wizard-pane.js';
+
+/**
+ * VlWizard
+ * @class
+ * @classdesc Gebruik de wizard om de gebruiker door een proces met meerdere stappen te begeleiden. Een wizard laat je toe om een complex proces op te splitsen in hapklare acties. Een wizard maakt het ook mogelijk om opties afhankelijk te maken van de keuzes in een voorgaande stap.
+ * 
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-wizard.html|Demo}
+ * 
+ */
+export class VlWizard extends VlElement(HTMLElement) {
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+
+                .vl-wizard__panes {
+                    overflow: hidden;
+                }
+            </style>
+            <section id="vl-wizard-id" class="vl-wizard" data-vl-wizard>
```

Comment:
id = vl-wizard(-section) misschien beter

Reply:
Id werd overgenomen van Webuniversum, wordt intern niet gebruikt maar kan potentieel wel gebruikt worden door de achterliggende code.

Reply:
Dat lijkt niet het geval te zijn, dus zal het weg doen. Indien dit id in de toekomst toch verwacht wordt, hebben we onze testen ter validatie.